store/src/java/com/zimbra/cs/gal/GalSearchControl.java

```diff
@@ -346,12 +346,15 @@ private void generateSearchQuery(Account galAcct) throws ServiceException {
     private String getSearchQuery(Account galAcct, boolean addInId) throws ServiceException {
         String query = mParams.getQuery();
         String searchByDn = mParams.getSearchEntryByDn();
+        String galSearchQuery = mParams.getGalSearchQuery();
```

Comment:
what's the difference between mParams.getQuery() & mParams.getGalSearchQuery(), is there an existing query param?

Reply:
So, SearchGal searches data on `name` request param, and mParams.getQuery(), holds the value for the same. Which searches inside the contact field.
And we want to search data inside any field, so we introduce a new parameter named `query`, and mParams.getGalSearchQuery() holds value for the same.